### PR TITLE
bug(CORE-462): fix url for sops

### PR DESCRIPTION
**Pull request by igajsin**
## Description:

Fix a URL to install sops.

**Closes Issues:**
-  https://jira.signavio.com/browse/CORE-462

### CODE REVIEW

**Review by dmitrybayanov — APPROVED** [review]

File: src/scripts/install-sops.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ Install() {
   if ! [ -x "$(command -v ${TOOL_NAME})" ]; then
     mkdir -p "${BIN_PATH}"
     if [ "${TOOL_VERSION}" = "latest" ]; then
-      TOOL_VERSION=$(curl -s https://api.github.com/repos/mozilla/sops/releases/latest | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
+      TOOL_VERSION=$(curl -s https://api.github.com/repositories/40684033/releases/latest | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
```
Comment (cruschke) [inline src/scripts/install-sops.sh:8]: what is this magic number? the code before looks more correct :-)

Comment (igajsin) [reply src/scripts/install-sops.sh:8]: I agree, that original URL looked better, but when you call it
```
 curl -s https://api.github.com/repos/mozilla/sops/releases/latest
{
  "message": "Moved Permanently",
  "url": "https://api.github.com/repositories/40684033/releases/latest",
  "documentation_url": "https://docs.github.com/v3/#http-redirects"
}
```

Comment (karlderkaefer) [reply src/scripts/install-sops.sh:8]: just -L for follow?

Comment (karlderkaefer) [reply src/scripts/install-sops.sh:8]: ah sops has different repo 
```suggestion
      TOOL_VERSION=$(curl -s https://api.github.com/repos/getsops/sops/releases/latest | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
```

**Pull request merged**